config/locales/en/course/assessment/submission/submissions.yml

```diff
@@ -39,6 +39,7 @@ en:
             auto_grade: 'Auto Grade Submission'
             evaluate_answers: 'Evaluate Answers'
             unsubmit: 'Unsubmit Submission'
+            unmark: 'Ungrade Submission'
```

Comment:
Should this be `Unpublish Submission` instead? As the grades will be kept as a draft.